views/partials/_whats-new.njk

```diff
@@ -0,0 +1,15 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<div class="app-whats-new">
+  <div class="app-width-container">
```

Comment:
This is just slight tidy up but we could place the markup inside this particular`.app-width-container` inside the `.app-width-container` below that the other main page content lives in, we don't necessarily need to create another `.app-width-container` here to do that.